jobs/setup-for-incremental-updates.yml

```diff
@@ -0,0 +1,45 @@
+# Template that takes in the following parameters
+# ${siteName}
+# ${partitionNum}
+# The purpose of this job is to refresh all of the DW table that are shared across multiple servers
+type: "job-pipeline"
+description: "Setting up to support incremental updates for ${siteName} using partition ${partitionNum}"
+configuration:
+  jobs:
```

Comment:
The way I'm setting this up, is I'm creating a new table in SQL Server that we can use to store the watermark values between executions of each job.  This will also give us the ability to review/monitor/sanity check the behavior we expect and the status of a given incremental loading job.  It will also, in theory, provide a mechanism to force an incremental job to reset - just delete all rows out of this table for a given table, and that table will be reloaded fully from scratch on the next execution.